### ISSUE TITLE
Add more resolutions w/out full removal/reinstall?

### ISSUE THREAD

**Issue by CorneliousJD** [ge9/IddSampleDriver#24]
I need to add some more resolutions but I tried to simply update the driver but Windows tells me it's already installed the best version.

Is there any way to update the the driver w/out full removal of the device and reinstall?
I ask because this generates a new display ID such as \\.\DISPLAY6 -- and I have scripts using it by name so I'd like to avoid having to update all of those.

I did try to change the iddsampledriver.inf file and update the DriverVer line to increase the date/number but no luck there, it still tells me I have the latest driver.

Any insight here is appreciated.

**Comment by ge9** [comment]
You can edit option.txt and then deactivate & activate the driver through "device manager", can't you?

**Comment by CorneliousJD** [comment]
Yes this does work but it changes the device name. for example the one I was using in all my scripts was \\.\DISPLAY6 and after doing that I can see/use the updated resolutions but it is now \\.\DISPLAY7

Is there any way to prevent this from receiving a new display name all the time? 
If at all possible it would be great to have flexibility to add/remove resolutions w/out the display ID changing, since I have to use that ID in various other scripts.

**Comment by XPRAMT** [comment]
> Yes this does work but it changes the device name. for example the one I was using in all my scripts was \.\DISPLAY6 and after doing that I can see/use the updated resolutions but it is now \.\DISPLAY7
> 
> Is there any way to prevent this from receiving a new display name all the time? If at all possible it would be great to have flexibility to add/remove resolutions w/out the display ID changing, since I have to use that ID in various other scripts.

I have the same question, the DISPLAY numbers are getting higher and higher

**Comment by SoFMeRight** [comment]
I know this is a little late but I do have an installer that should be compatible with this driver. I use a different fork but glad to share with anyone who can benefit. It's 100% in autohotkey, the source is included as well. It also supports CMD/CLI syntax and can be integrated into Sunshine directly by following a straightforward guide.

[Virtual Display Driver: Wizard](https://github.com/sofmeright/IDDSampleDriver_Wizard)
A GUI tool that can integrate with other software such as Sunshine for efficient manipulation (Install / Uninstall / Reload / Configure) of Indirect Display Driver Sample (IddSample) implementations like the HDR capable [Virtual Display Driver from itsmikethetech](https://github.com/itsmikethetech/Virtual-Display-Driver) that works on Windows 10/11. This installer/configuration tool is intended to provide a more elegant solution in place of needing to edit configs or use Device Manger for driver installation.
